js/bundle.go

```diff
@@ -55,16 +56,32 @@ type BundleInstance struct {
 	Default goja.Callable
 }
 
+type cacheOnReadFs struct {
+	afero.Fs
+	cache afero.Fs
+}
+
+func newCacheOnReadFs(base, layer afero.Fs, cacheTime time.Duration) afero.Fs {
+	return cacheOnReadFs{
+		Fs:    afero.NewCacheOnReadFs(base, layer, cacheTime),
+		cache: layer,
+	}
+}
+
+func (c cacheOnReadFs) GetCachedFs() afero.Fs {
+	return c.cache
+}
+
 // NewBundle creates a new bundle from a source file and a filesystem.
-func NewBundle(src *lib.SourceData, fs afero.Fs, rtOpts lib.RuntimeOptions) (*Bundle, error) {
+func NewBundle(src *lib.SourceData, fileFS afero.Fs, rtOpts lib.RuntimeOptions) (*Bundle, error) {
```

Comment:
In my current reworking due to windows ... I now provide the whole `map[string]afero.Fs`, for reasons that will become apparent :) 
But yeah the idea here was that this would be the fileFs that was used to load the original script from ... Obviously this doesn't work very well as 
1. We won't cache that one when we first load it in the `cmd/run.go` or elsewhere
2. Won't work if we load (and again not cache it) if it was from a remote script ... aka https Fs :)